controllers/vrg_volrep.go

```diff
@@ -1762,32 +1762,38 @@ func (v *VRGInstance) validatePVExistence(pv *corev1.PersistentVolume) error {
 		return fmt.Errorf("failed to get PV (%s) (%w)", existingPV.GetName(), err)
 	}
 
-	if existingPV.ObjectMeta.Annotations != nil &&
-		existingPV.ObjectMeta.Annotations[PVRestoreAnnotation] == "True" {
-		// Should we check and see if PV in being deleted? Should we just treat it as exists
-		// and then we don't care if deletion takes place later, which is what we do now?
-		v.log.Info("PV exists and managed by Ramen", "PV", existingPV.GetName())
-
-		return nil
-	}
-
+	// If the PV is bound and matches with the PV we would have restored then return now
 	if v.pvMatches(existingPV, pv) {
```

Comment:
There is no way to differentiate between a Ramen restored PV and initially created PV in MDR as the PV is never deleted. The only error case is if the PV is bound already to a PVC with a different name. Hence the skip on that check.

Reply:
I was trying to say that you can have a PV in cluster 2 with the same name and claimRef.  Then you failover to cluster 2, `pvMatches` fine, but Ramen didn't restore the PV. This scenario is unlikely, but there is a small possibility that one can create a test app in cluster 2 with the same PVC name/namespace, then deletes it, but retains the PV.

Reply:
Yes, a PV left over from a previous app installation will also be taken over in this case. I think one way to prevent that is to place a unique drpc/vrg annotation on the pv when we restore it and not take over the PV if this annotation differs. I will file a new PR for that.